src/containers/SearchPage/components/form/SearchContentForm.tsx

```diff
@@ -215,6 +216,34 @@ class SearchContentForm extends Component<Props & WithTranslation, State> {
       },
     ];
 
+    const tagTypes = [
```

Comment:
Som en følge av https://github.com/NDLANO/editorial-frontend/pull/1138 må language også være med for content. Så denne pr må vente til den andre er merga.